edge-apps/tfl-bus-status/README.md

```diff
@@ -0,0 +1,64 @@
+# TFL Bus Information Edge App
+
+This guide will walk you through the steps to create an TFL Bus Status Edge App in Screenly.
+
+
+## Prerequisites
+
+* Install Screenly CLI - Please follow the guide [here](https://github.com/Screenly/cli)
+
+## Steps
+
+Step 1. **Log in to the Screenly account via CLI**
+
+Follow the on-screen instructions to log in to your Screenly account.
+
+`$ screenly login`
+
+Step 2. **Download and Open Edge App Playground**
+
+`$ git clone https://github.com/Screenly/Playground.git`
+
+Step 3. **Enter tfl-bus-status Edge App Directory**
+
+`$ cd edge-apps/tfl-bus-status`
+
+Step 4. **Create a New tfl-bus-status Edge App:**
+
+`$ screenly edge-app create --name TFL_EdgeApp --in-place`
```

Comment:
Nit but `$ screenly edge-app create --name "TFL Edge App" --in-place` probably looks better.

Reply:
Thanks. Updating.